lib/dependency_validator.dart

```diff
@@ -112,12 +123,19 @@ void run({
     ..addAll(listScssFilesIn('test/', excludedDirs))
     ..addAll(listScssFilesIn('tool/', excludedDirs))
     ..addAll(listScssFilesIn('web/', excludedDirs));
+  final nonLibLessFiles = <File>[]
+    ..addAll(listScssFilesIn('example/', excludedDirs))
+    ..addAll(listScssFilesIn('test/', excludedDirs))
+    ..addAll(listScssFilesIn('tool/', excludedDirs))
+    ..addAll(listScssFilesIn('web/', excludedDirs));
```

Comment:
Should these be calling `listLessFilesIn()`?
```suggestion
    ..addAll(listLessFilesIn('example/', excludedDirs))
    ..addAll(listLessFilesIn('test/', excludedDirs))
    ..addAll(listLessFilesIn('tool/', excludedDirs))
    ..addAll(listLessFilesIn('web/', excludedDirs));
```